app/frontend/src/features/search/SearchResultsList.tsx

```diff
@@ -21,21 +21,26 @@ import hasAtLeastOnePage from "utils/hasAtLeastOnePage";
 const useStyles = makeStyles((theme) => ({
   mapResults: {
     height: "15rem",
-    zIndex: 3,
-    overflow: "visible",
+    overflowY: "auto",
+    backgroundColor: theme.palette.background.default,
     [theme.breakpoints.up("md")]: {
       height: "auto",
-      width: "35%",
-      overflow: "auto",
+      width: "30rem",
+      padding: theme.spacing(3),
     },
   },
   baseMargin: { margin: theme.spacing(2) },
   searchDesktop: {
-    margin: theme.spacing(0, 2),
-    marginTop: theme.spacing(2),
+    "& > div > *": {
+      flexShrink: 0,
+      width: "100%",
+    },
```

Comment:
The field should accept a `fullWidth` prop which is probably clearer than doing this